methods.py

```diff
@@ -1473,6 +1484,27 @@ def format_key_value(v):
         sys.exit()
 
 
+############################################################
+# FILE GENERATION & FORMATTING
+############################################################
+
+
+NodeOrPath = Union[str, "Node"]
+
+
+def node_to_path(value: NodeOrPath) -> str:
+    """
+    Converts a raw string / SCons Node to a pure string path, Regardless of what
+    type is passed in, the value is normalized & assigned posix separators.
+    """
+    if not isinstance(value, str):
+        if not hasattr(value, "get_abspath"):
+            raise TypeError(f'Expected type "str" or "Node"; was passed {type(value)}.')
+        value = value.get_abspath()
+
+    return os.path.normpath(str(value)).replace("\\", "/")
```

Comment:
I think in this case we should start moving to `posixpath` python module which is generally normal `pathlib` but works with `posix` paths regardless of which host system is used